api/authentication/authentication.go

```diff
@@ -302,15 +302,22 @@ func (config authenticatorRESTConfigurator) verifyJWT(c *gin.Context) {
 }
 
 func getTokenFromHeader(c *gin.Context) (string, error) {
-	t := c.Request.Header.Get("Authorization")
+	type authHeader struct {
```

Comment:
Let's move this definition to be outside of the function. Definitions are commonly place in the global scope.